lib_expense/src/test/java/com/example/expenselogger/lib_expense/domain/usecase/DeleteExpenseTest.kt

```diff
@@ -23,4 +26,13 @@ public class DeleteExpenseTest {
             deleteExpense.invoke(expense)
             assertThat(expenseContract.getExpenses().size).isEqualTo(0)
         }
+
+    @Test
+    public fun `verify that deleteExpense usecase calls expenseContract`(): Unit = runBlockingTest {
```

Comment:
Test name leans too much into the implementation of DeleteExpense
Try this instead. 
'when delete expense is called, then expense is deleted'
That way if something changes in the class's implementation , then you don't have to redo the test name

Do this for all the other tests you wrote

Reply:
Ohh... so it is just better for the test name to show what the method does, but not how it does it?

Reply:
Exactly

Reply:
Ohhh okay. Thank you!!